app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,15 @@
+from abc import ABC
+
+from app.players.player import Player
+
+
+class Dwarf(Player, ABC):
+    def __init__(
+            self,
+            nickname: str,
+            favourite_dish: str) -> None:

```

Comment:
One attribute per line, when you have a lot of attributes. At checklist you have a good example of styling
```    
    def long_function_name(
            var_one,
            var_two,
            var_three,
            var_four
    ) -> None:
```